.github/workflows/macos-checks.yml

```diff
@@ -44,7 +46,7 @@ jobs:
         key: ${{ runner.os }}-mint-${{ hashFiles('./Mintfile') }}
         restore-keys: ${{ runner.os }}-mint-
     - name: Run bootstrap.sh
-      run: ./bootstrap.sh
+      run: ./Scripts/bootstrap.sh
```

Comment:
We did not catch that before because this workflow was disabled.
Because I updated the yml, this was automatically enabled again, and I spot this issue.

It is disabled again now